sandbox/sandbox.go

```diff
@@ -1,3 +1,5 @@
+// +build !windows
```

Comment:
@jhowardmsft I agree with @mrjana. The main reason for having a generic Sandbox interface and platform specific implementation is to provide just that support. Its perfectly understandable that the implementation of that API interface in windows could be different (or even be absent for the fore-seeable future). But that detail must be kept on the implementation and keep the API free of platform specific details.

Yes, without a proper CI, Linux specific implementation commits are inadvertently breaking other platforms. As Maintainers we will be extra cautious. And infact push harder to have a good CI in place for cross-platform testing.
I think that will solve the nagging issue that you are facing.